src/components/dms/ConvoMenu.tsx

```diff
@@ -28,16 +29,15 @@ import * as Prompt from '#/components/Prompt'
 import {Bubble_Stroke2_Corner2_Rounded as Bubble} from '../icons/Bubble'
 
 let ConvoMenu = ({
-  convo,
+  convoIdOrConvo,
   profile,
-  onUpdateConvo,
   control,
   currentScreen,
   showMarkAsRead,
   hideTrigger,
   triggerOpacity,
 }: {
-  convo: ChatBskyConvoDefs.ConvoView
+  convoIdOrConvo: string | ConvoView
```

Comment:
Do we need to accept both? If not, let's just use one or the other, preferably `id` I think.

Reply:
Probably will never happen, but i left it because do we ever anticipate it for something like viewing a report? We'd have to load it first. We actually do this with RQ in other areas as well fwiw.

Reply:
I'm not sure I follow, seems like `ConvoMenu` should only be used in contexts where the convo is already loaded either way. And if we need to rely on server state, then fetching it within the component is probably the right call. Otherwise if we pass in a `ConvoView`, how do we know that it's revalidating on mutate?

It's a small thing, but I don't think we should build loose contracts from the get-go if we can help it.

This isn't a blocker though imo ✌️

Reply:
Right for sure. There is sense in doing that in the overall list - you have access to `ConvoListChats` (or...whatever the type is I forget so many names!) and can just pass in each value when you render the list.

The problem is when we're in an individual chat screen. We have `convo` from the chat service, but unless we add `mute()` to the chat service then we have to keep track of the mute state some other way.

```ts
const [muted, setMuted] = useState(convo.muted)

const onMutedChange = (muted) => setMuted(muted)

// Return dialog that calls onMutedChange
```

This to me feels more dangerous, since we definitely are not going to know of any changes that happen elsewhere. Even if we mutate with `setQueryData` or invalidate queries with `invalidateQuery`, our mute state in _this_ screen still wouldn't be updated.

Reply:
Ok I see what you're saying. Yeah like I mentioned below maybe this needs to move into `Convo`.

Reply:
Will mark this one as a draft for now pending what we end up doing 👍